using-mobile-extensions/mobile-core/mobile-core-api-reference.md

```diff
@@ -713,7 +713,7 @@ MobileCore.getSdkIdentities { (content, error) in
 This is a generic API to request each extension to reset the identities it owns. Each extension can decide whether and how it will reponse to this API call.
 
 {% hint style="warning" %}
-This API is only handled by the [Edge](../adobe-edge/README) and [Edge Identity](../adobe-edge-identity/README.md) extensions. Calling `resetIdentities` does not clear identities in non-edge extensions, such as [Identity](identity/README).
+This API is only handled by the [Edge](https://aep-sdks.gitbook.io/docs/v/AEP-Edge-Docs/using-mobile-extensions/adobe-edge) and [Edge Identity](https://aep-sdks.gitbook.io/docs/v/AEP-Edge-Docs/using-mobile-extensions/adobe-edge-identity) extensions. Calling `resetIdentities` does not clear identities in non-edge extensions, such as [Identity](https://aep-sdks.gitbook.io/docs/v/AEP-Edge-Docs/using-mobile-extensions/mobile-core/identity).
```

Comment:
Why can't we use relative links still? when we will switch from AEP-Edge-Docs -> main all these links may break